buddyboss/buddyboss-platform
# Fixed - Participant avatar is clickable but participant name is not c…

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2066.

### How to test the changes in this Pull Request:

1. Go to Messages
2. Open the existing conversation
3. Participant name should clickable now

### Proof Screenshots or Video

https://www.loom.com/share/04024a601a5448fd802b367b5f4a5df1


### Other information:
